cms/templates/settings.html

```diff
@@ -361,7 +361,7 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
               </ol>
             </section>
 
-          % if about_page_editable:
+          % if about_page_editable or is_prerequisite_courses_enabled or settings.FEATURES.get('ENTRANCE_EXAMS'):
```

Comment:
In addition we should update the entrance exam settings reference on line 394 to match the new field ref